.Lib9c.Tools/SubCommand/Market.cs

```diff
@@ -0,0 +1,189 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Cocona;
+using Libplanet;
+using Libplanet.Assets;
+using Libplanet.Blockchain;
+using Libplanet.Blocks;
+using Libplanet.RocksDBStore;
+using Libplanet.Store;
+using Libplanet.Tx;
+using Nekoyume.Action;
+using Nekoyume.Model.Item;
+using Serilog.Core;
+using NCAction = Libplanet.Action.PolymorphicAction<Nekoyume.Action.ActionBase>;
+
+namespace Lib9c.Tools.SubCommand
+{
+    public class Market
+    {
+        [Command(Description = "Query market transactions.")]
+        public void Query(
+            [Option('v', Description = "Print more logs.")]
+            bool verbose,
+            [Option('s', Description = "Path to store.")]
```

Comment:
This message seems not to present its purpose well. The argument `storePath` has been being used to open `IStore` implementation. I felt it like *"the path to store something"* (`Noun to Verb` not `Noun to Noun`) 🤔 

But I'm not sure about English grammer so feel free to say *"this description is correct because ..."* if I was wrong 😅 

Don't care too much 🙏🏻